aeron-client/src/main/cpp/ClientConductor.h

```diff
@@ -32,6 +32,7 @@
 #include "LogBuffers.h"
 #include "HeartbeatTimestamp.h"
 #include "util/Export.h"
+#include "AeronVersion.h"
```

Comment:
Do we need to implement this in the wrapper as well?

Reply:
@mikeb01 We can do for symmetry but the wrapper is delegating to the C client conductor which means that the latter is going to use the `aeron_version.h` anyhow.